libs/plausible/src/lib/webEvents.ts

```diff
@@ -39,3 +39,50 @@ export const webMenuButtonClicked = () => {
   }
   plausibleCustomEvent(event)
 }
+
+export const haskolanamFilterClicked = (category: string, value: string) => {
+  const event: BaseEvent = {
+    eventName: `haskolanam.filter`,
+    featureName: '',
```

Comment:
I'm not familiar with how we use plausible, but do this new events not require featureName?

Maybe @RunarVestmann knows more?

Reply:
Working with plausible for the first time but my understanding is that it just prefixes your events (see web Login.., web Menu in plausible, and built in "Visit" features) I figured I'd just prefix with dot notation since "haskolanam" is not really a feature imo. If anyone has deeper knowledge or a particular way to structure plausible goals let me know.
![image](https://github.com/island-is/island.is/assets/22780994/37cd2532-599e-4f37-b55d-57b374de6e4e)